dashboard/test/ui/features/gamelab/eyes.feature

```diff
@@ -0,0 +1,16 @@
+@eyes
+@dashboard_db_access
+@as_student
```

Comment:
May need as_admin at some point in the near future, when we lock down gamelab again.  I'm working on this, could make this change myself when it's necessary.